client/src/pages/TourPage/TourPage.js

```diff
@@ -136,7 +110,7 @@ const FloatingDialog = styled(Box)(({ theme }) => ({
 }));
 
 const TourPage = () => {
-  const matchesMediaQuery = useMediaQuery('(min-width: 1051px)');
+  const screenWidth = useMediaQuery('(min-width: 1051px)');
```

Comment:
as for me, it is not understandable name for boolean variable

Reply:
i told you before, when you name the variable, you dont name it according to the data type  but to what content it holds for the sake of code readability

Reply:
it holds true or false so that it should tell rather it matches media query or not